addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/handler/KonnectedHandler.java

```diff
@@ -20,6 +20,7 @@
 import javax.measure.quantity.Dimensionless;
 import javax.measure.quantity.Temperature;
 
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
Please remove references to `NonNull`, since we are using `NonNullByDefault`, see http://www.eclipse.org/smarthome/documentation/development/conventions.html#null-annotations

Reply:
done